docs/running/cliOptions.rst

```diff
@@ -89,8 +89,8 @@ about the performance of jobs.
                         directory. Temp files and folders will be placed in a
                         directory ``toil-<workflowID>`` within workDir. The
                         workflowID is generated by Toil and will be reported in
-                        the workflow logs. Default is determined by the
-                        variables (TMPDIR, TEMP, TMP) via mkdtemp. This
+                        the workflow logs. Default is the temporary output
+                        directory (see ``--tmp-outdir-prefix``). This
```

Comment:
There's only a notion of `--tmp-outdir-prefix` in the CWL runner; Python workflows an the WDL runner don't have it, and this documentation is meant to cover all three. So this probably needs to say that for CWL workflows the default `--workDir` comes from `--tmp-outdir-prefix`, but for other cases it still comes from `mkdtemp`.

Also, the help that Toil actually prints is [here](https://github.com/DataBiosphere/toil/blob/303a8d8e518ed7d3aaaa08c7b8388501e39577e2/src/toil/options/common.py#L335-L345), and it also has the material about `mkdtemp`, so if in CWL mode that doesn't apply, we might want to change that text too. There we have access to a boolean `cwl` flag, so we could describe the CWL-related behavior when it is set and the non-CWL behavior otherwise.

Reply:
I reverted the change to the original text, and added a line about CWL workflows and `--tmp-outdir-prefix`. I don't know if you want it to be more elaborate.

I didn't touch the help that Toil prints. It was not completely clear to me where these changes had to be made, taking the `cwl` boolean into account.